README.md

```diff
@@ -1,66 +1,39 @@
-HTTP/gRPC/thrift/tcp/etc proxy, and can be deployed alongside existing
-applications regardless of what language they're written in. It works with many
-common protocols and utilizes Kubernetes as a backend for service discovery.
+Linkerd is a Cloud Native Computing Foundation ([CNCF][cncf]) project.
 
-It is separated into two major components: the control plane and the data plane.
-The control plane interacts with the service discovery backend, orchestrates the
-data plane and is written in [Go][golang]. The data plane runs alongside
-existing applications, provides the proxy that manages traffic itself and is
-written in [Rust][rust].
+## Repo layout
 
-Currently, Linkerd2 is capable of proxying all TCP traffic, including WebSockets
-and HTTP tunneling, along with reporting top-line metrics (success rates,
-latencies, etc) for all HTTP, HTTP/2, and gRPC traffic.
+This is the primary repo for the Linkerd 2.x line of development.
 
-Linkerd is hosted by the Cloud Native Computing Foundation ([CNCF][cncf]).
+The complete list of Linkerd repos is:
+* [linkerd2][linkerd2]: Main Linkerd 2.x repo, including control plane and CLI
+* [linkerd2-proxy][proxy]: Linkerd 2.x data plane proxy
+* [linkerd2-proxy-api][proxy-api]: Linkerd 2.x gRPC API bindings
+* [linkerd][linkerd1]: Linkerd 1.x
+* [linkerd-website][linkerd-website]: linkerd.io website (including docs for 1.x and 2.x)
+* [linkerd-examples][linkerd-examples]: example configs and apps
 
-## Get involved
-
-* [Users mailing list][linkerd-users]: Linkerd2 user discussion mailing
-  list.
-* [Developers mailing list][linkerd-dev]: Linkerd2 development discussion
-  mailing list.
-* [Announcements mailing list][linkerd-announce]: Linkerd2 announcements only
-  (low volume).
-* Follow [@linkerd][twitter] on Twitter.
-* Join the #linkerd2 channel on the [Linkerd Slack][slack].
-
-## Documentation
-
-View [Linkerd2 docs][linkerd-docs] for a more comprehensive guide to getting
-started, or use the instructions below.
-
-The doc source code is available in the [website][linkerd-website] repo.
-
-## Getting started with Linkerd2
-
-1. Install the Linkerd2 CLI with `curl https://run.linkerd.io/install | sh`.
-
-1. Add `$HOME/.linkerd2/bin` to your `PATH`.
-
-1. Install Linkerd2 into your Kubernetes cluster with `linkerd install | kubectl
-   apply -f -`.
-
-1. Verify that the installation succeeded with `linkerd check`.
+## Quickstart and documentation
 
-1. Explore the Linkerd2 controller dashboard with `linkerd dashboard`.
+You can run Linkerd on any Kubernetes 1.9+ cluster in a matter of seconds. See
+the [Linkerd Getting Started Guide][getting-started] for now.
 
-1. Optionally, install a [demo application][linkerd-demo] to run with Linkerd2.
-
-1. Add [your own service][linkerd-inject] to the Linkerd2 mesh!
+View the [Linkerd docs][linkerd-docs] for a more comprehensive guide to getting
+started, or use the instructions below. The doc source code is available in the
```

Comment:
There are no longer instructions in this README, so "use the instruction below" doesn't make sense here. I'd change to:

> View the Linkerd docs for a more comprehensive guide to getting started.